cicecore/cicedynB/general/ice_forcing.F90

```diff
@@ -1626,12 +1629,20 @@ subroutine prepare_forcing (nx_block, ny_block, &
 
       endif
 
-      if (calc_strair) then
+      if (calc_strair .and. (.not.rotate_wind)) then
```

Comment:
The logic here is hard to follow.  Maybe better like this?  
```
if (calc_strair) then
  if (rotate_wind) then
     ...
  else ! calc_strair and wind not rotated
     ...
  endif
else ! .not. calc_strair
  if (rotate_wind) then
     ...
  else ! not calc_strair and wind not rotated - not currently implemented
     ...
  endif
endif
```

Reply:
Agree, It is easier to read splitting up like you suggested despite of only 3 possible outcome ... today. I have repeated the do-loops to avoid if-sentence inside loops.